src/net/net.go

```diff
@@ -7,10 +7,19 @@
 package net
 
 import (
-	"io"
 	"time"
+
+	"tinygo.org/x/drivers/netdev"
```

Comment:
I think maybe this shouldn't be referencing the drivers module; it might be preferable to have a copy of the netdev.Netdever interface in TinyGo instead of in `drivers` (or alternatively, have the primary definition be in TinyGo and referenced from `drivers`).

Reply:
I had it that way at one point, but changed it as to not add any new exports to "net" package.

Reply:
I understand the motivation, but we for sure do not wish for the circular reference. Either use a subpackage, just add another file with the exported interface, or copy the interface definition into both the tinygo repo and also a duplicate the drivers package.

Reply:
Ok, new commit moving drivers/netdev to tinygo/src/net/netdev.  This should decouple "net" and "netdev" and remove the circular reference.

Reply:
If we do want to have netdev package inside of `Tinygo` it should be unreachable by external users. You can do this by putting `netdev` directory under another directory called `internal`. See [Dave Cheney's](https://dave.cheney.net/2019/10/06/use-internal-packages-to-reduce-your-public-api-surface) blog post.